kitchen.yml

```diff
@@ -75,17 +83,4 @@ suites:
     attributes:
     includes:
       - amazon2
-      - amazon2-2018
       - sles-15
-  - name: windows
```

Comment:
The windows suite was never supported anyway. It it less confusing to get rid of it.